etc/dehydrated/hook.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+CONSUL_ROOT="http://localhost:8500/v1/kv/nginx"
+CHALLENGE_PATH="/.well-known/acme-challenge"
+
+function deploy_challenge {
+    local DOMAIN="${1}" TOKEN_FILENAME="${2}" TOKEN_VALUE="${3}"
+
+    curl -sX PUT -d "${TOKEN_FILENAME}" ${CONSUL_ROOT}/domains/${DOMAIN}/acme/challenge/token-filename > /dev/null
```

Comment:
Agreed. I actually put those there during development because they were spamming `true` to STDOUT and cluttering up other stuff I was trying to read. You'd hope alarms would go to STDERR, but I know that's not always the case.